bitcoin/embedded/Cargo.toml

```diff
@@ -27,3 +27,6 @@ bench = false
 codegen-units = 1 # better optimizations
 debug = true # symbols are nice and they don't increase the size on Flash
 lto = true # better optimizations
+
+[patch.crates-io]
+bitcoin_hashes = { path = "../../hashes" }
```

Comment:
This change is worth noting, as we crate smash this will happen again. `embedded` and `fuzz` crates are explicitly excluded from the workspace so the workspace manifest `patch` section does not apply to them.

Reply:
Had to do the same in `test.sh` for `dep_test`.